fedimint-core/src/config.rs

```diff
@@ -321,7 +322,35 @@ impl<M> ModuleGenRegistry<M> {
     where
         M: Clone,
     {
-        for hardcoded_module in ["mint", "ln", "wallet"] {
+        let args: Vec<String> = env::args().collect();
```

Comment:
We should parse these via `clap`, and pass here as an argument.

Reply:
`fedimint-server` already takes care of passing the args, BTW. We can just stick it there, and then just need to carry it to here.

Reply:
Oh! Let me do that again. Thanks.